build-tools/xaprepare/xaprepare/Application/Utilities.cs

```diff
@@ -99,7 +99,10 @@ public static async Task<bool> BuildRemapRef (Context context, bool haveManagedR
 			bool result = await msbuild.Run (
 				projectPath: projectPath,
 				logTag: "remap-assembly-ref",
-				binlogName: "build-remap-assembly-ref"
+				arguments: new List<string> {
+					"/restore",
```

Comment:
I think it's better to add `/restore` to `MSBuildRunner.StandardArguments`, seeing that we will be needing the parameter everywhere.